lib/themes/dosomething/paraneue_dosomething/templates/system/partials/navigation.tpl.php

```diff
@@ -35,7 +35,7 @@
           <a href="#" class="navigation__dropdown-toggle"><?php print $user_identifier ?></a>
           <ul>
             <li><?php print l("My Account", 'user/'. $user->uid); ?></li>
-            <li><a id="link--logout" href="<?php print $front_page; ?>user/logout"><?php print t('Log Out'); ?></a></li>
+            <li><a id="link--logout" href="<?php print $GLOBALS['base_url'] ?>/user/logout"><?php print t('Log Out'); ?></a></li>
```

Comment:
I think we should be using l() here instead of printing out each part of the url.